### PR TITLE
Incidental cleanup pass #1

### PR DESCRIPTION
**Pull request by dracontes**
- Cleaning up `K:etbCounter` lines
○ `no condition` corrected to `no Condition`
○ Moved Crowd-Control Warden's description to a more appropriate place after testing.
○ Banquet Guests and Eternity Vessel had the relevant description added.
○ Removed the description from Swarm Shambler as it's adequately done via automatic text.
- Various typos and other editing lapses, not the least of which were instances of missing `$` on parameters.

### CODE REVIEW

File: forge-gui/res/cardsfolder/c/crowd_control_warden.txt [hunk 1]
```diff
@@ -3,8 +3,8 @@ ManaCost:3 G W
 Types:Creature Centaur Soldier
 PT:4/4
 K:Disguise:3 GW GW
-K:etbCounter:P1P1:X
-R:Event$ TurnFaceUp | ValidCard$ Card.Self | ReplaceWith$ AddCounters | ActiveZones$ Battlefield | Description$ As CARDNAME enters the battlefield or is turned face up, put X +1/+1 counters on it, where X is the number of other creatures you control.
+K:etbCounter:P1P1:X:no Condition:As CARDNAME enters the battlefield or is turned face up, put X +1/+1 counters on it, where X is the number of other creatures you control.
+R:Event$ TurnFaceUp | ValidCard$ Card.Self | ReplaceWith$ AddCounters | ActiveZones$ Battlefield
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/c/crowd_control_warden.txt:7]: keep the Description on the Replacement and Mark it as `Secondary$ True`

File: forge-gui/res/cardsfolder/b/balduvian_warlord.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ ManaCost:3 R
 Types:Creature Human Barbarian
 PT:3/2
 A:AB$ RemoveFromCombat | Cost$ T | ActivationPhases$ Declare Blockers | ValidTgts$ Creature.blocking | Defined$ Targeted | UnblockCreaturesBlockedOnlyBy$ Targeted | SubAbility$ ChooseAttacker | SpellDescription$ Remove target blocking creature from combat. Creatures it was blocking that hadn't become blocked by another creature this combat become unblocked.
-SVar:ChooseAttacker:DB$ ChooseCard | Defined$ You | Choices$ Creature.attacking | ChoiceTitle$ Choose an attacker to block | Mandatory$ True | SubAbility$ Block | SpellDescriptions$ None | StackDescription$ None
+SVar:ChooseAttacker:DB$ ChooseCard | Defined$ You | Choices$ Creature.attacking | ChoiceTitle$ Choose an attacker to block | Mandatory$ True | SubAbility$ Block | SpellDescription$ None | StackDescription$ None
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/b/balduvian_warlord.txt:6]: bad change:
image

File: forge-gui/res/cardsfolder/t/triarch_stalker.txt [hunk 3]
```diff
@@ -3,7 +3,7 @@ ManaCost:3 B B
 Types:Artifact Creature Necron
 PT:4/5
 S:Mode$ Continuous | Affected$ Creature.attacking ChosenPlayer | AddKeyword$ Menace | Description$ Creatures attacking the last chosen player have menace.
-T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | Execute$ TrigChoose | TriggerZones$ Battlefield | TriggerDescriptionLure the Unwary — At the beginning of combat on your turn, choose an opponent.
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | Execute$ TrigChoose | TriggerZones$ Battlefield | TriggerDescription$ Lure the Unwary — At the beginning of combat on your turn, choose an opponent.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/t/triarch_stalker.txt:6]: intro part wrong

File: forge-gui/res/cardsfolder/b/balduvian_warlord.txt (hunk 2 above)
Comment (dracontes) [reply forge-gui/res/cardsfolder/b/balduvian_warlord.txt]: Oops. Now I recall that `SpellDescriptions$` can be removed without much issue. While I was at it, I added an explicit `TgtPrompt$`.

**Review by tool4ever — APPROVED** [review]

**Pull request merged**